R/validation.R

```diff
@@ -0,0 +1,127 @@
+stop_if_missing <- function(serosurvey, must_have_cols) {
+  if (
+    !all(
+      must_have_cols
+      %in% colnames(serosurvey)
+    )
+  ) {
+    missing <-
+      must_have_cols[which(!(must_have_cols %in% colnames(serosurvey)))]
+    stop(
+      "The following mandatory columns in `serosurvey` are missing.\n",
+      toString(missing)
+    )
+  }
+}
```

Comment:
Is there a feature difference between this and `checkmate::assert_names(must.include =)`?